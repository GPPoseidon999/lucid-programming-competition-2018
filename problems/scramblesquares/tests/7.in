4 7
4 3 1 3
1 3 0 2
7 0 6 0
2 4 8 6
3 2 6 3
3 0 7 4
3 4 2 0
9 5 3 4
2 0 3 7
0 0 0 0
9 1 1 0
8 7 5 1
3 0 5 7
9 4 7 4
6 6 9 3
7 7 4 6
6 1 9 4
6 4 2 2
7 4 9 4
9 2 7 0
0 4 6 5
1 7 4 4
5 8 0 7
5 9 3 8
6 4 6 6
4 0 4 4
0 0 0 0
3 3 3 0
