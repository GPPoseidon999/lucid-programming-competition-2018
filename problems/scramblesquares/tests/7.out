\ 3 /\ 3 /\ 9 /\ 6 /
4 X 00 X 44 X 44 X 1
/ 7 \/ 2 \/ 7 \/ 9 \
\ 7 /\ 2 /\ 7 /\ 9 /
6 X 77 X 00 X 00 X 2
/ 4 \/ 3 \/ 6 \/ 7 \
\ 4 /\ 3 /\ 6 /\ 7 /
3 X 33 X 22 X 44 X 4
/ 1 \/ 6 \/ 2 \/ 9 \
\ 1 /\ 6 /\ 2 /\ 9 /
2 X 33 X 66 X 44 X 5
/ 0 \/ 9 \/ 8 \/ 3 \
\ 0 /\ 9 /\ 8 /\ 3 /
0 X 00 X 11 X 77 X 0
/ 0 \/ 1 \/ 5 \/ 5 \
\ 0 /\ 1 /\ 5 /\ 5 /
5 X 44 X 77 X 88 X 9
/ 6 \/ 4 \/ 0 \/ 3 \
\ 6 /\ 4 /\ 0 /\ 3 /
6 X 44 X 00 X 00 X 3
/ 6 \/ 4 \/ 0 \/ 3 \
